Sources/Subscription/StoreSubscriptionConfiguration.swift

```diff
@@ -45,8 +53,8 @@ final class DefaultStoreSubscriptionConfiguration: StoreSubscriptionConfiguratio
                   environment: .staging,
                   identifiersByRegion: [.usa: ["ios.subscription.1month",
                                                "ios.subscription.1year",
-                                               "ios.subscription.1month.freetrial.dev",
-                                               "ios.subscription.1year.freetrial.dev"],
+                                               "ios.subscription.1month\(StoreSubscriptionConstants.freeTrialSuffix).dev",
+                                               "ios.subscription.1year\(StoreSubscriptionConstants.freeTrialSuffix).dev"],
```

Comment:
Why the `.dev` part? Couldn't we reuse the original names and append it with suffix? The multitude of subscriptions is not helpful so sticking to some consistency can be helpful.

Reply:
(Chatted with @miasma13 on MM). The staging subscriptions here are not ideally named. However, as the production subs too follow convention, we are ok with leaving this for now.